browsing-context.md

```diff
@@ -240,23 +183,37 @@ if (geoPermission.state === "denied") {
 
 _Note: the above code only makes sense if we decide that permissions are denied in prerendering browsing contexts, instead of having them hang until activation. [That plan](#restrictions-on-the-basis-of-being-non-user-visible) is still tentative._
 
-Finally, there's the case of the [page visibility API](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API). Our current plan is to treat (non-portal) prerendering browsing contexts as hidden, until activation, in which case code that only wants to run upon the user viewing the page could be done as follows:
+Finally, for cases related to rendering behavior, we propose a [dedicated prerendering state API](./prerendering-state.md):
 
 ```js
-if (document.visibilityState === "visible") {
-    videoElement.play();
+function afterPrerendering() {
+  // grab user data from cookies/IndexedDB
```

Comment:
This and requesting notifications access seems better covered by the storage and permissions APIs above. More relevant to prerendering could be "start a video/animation", "fetch large resources", "connect to chat server", etc.

Reply:
Good call; leftover from some copy and pasting!